### PR TITLE
GEF service removal

### PR DESCRIPTION
**Pull request by ubuntolog**
This pull request adds the service removal option and closes the issue https://github.com/EUDAT-GEF/GEF/issues/156

Services are not actually removed: they are marked as removed and the system do not display them

### CODE REVIEW

File: webui/src/components/Service.jsx [hunk 1]
```diff
@@ -273,6 +277,9 @@ class Service extends React.Component {
                             <button type="submit" className="btn btn-default" onClick={this.handleModalOpen.bind(this)}>
                                 <span className="glyphicon glyphicon-edit" aria-hidden="true"></span> Edit Metadata
                             </button>
+                            <button type="submit" className="btn btn-default" onClick={this.handleRemoveService.bind(this)}>
```
Comment (emanueldima) [inline webui/src/components/Service.jsx]: The button type here and above should not be 'submit', but 'button', as there is no form involved.

File: webui/src/components/Services.jsx [hunk 2]
```diff
@@ -35,7 +35,9 @@ const Header = () => (
 class Services extends React.Component {
     constructor(props) {
         super(props);
+        console.log(this.props);
```
Comment (emanueldima) [inline webui/src/components/Services.jsx]: remove?

Comment (ubuntolog) [reply webui/src/components/Services.jsx]: yes, it was used for debugging

File: webui/src/components/Service.jsx (hunk 1 above)
Comment (ubuntolog) [reply webui/src/components/Service.jsx]: right

**Pull request merged**